### PR TITLE
Update polkassembly.ts networks

### PR DESCRIPTION
**Pull request by KanishkaRajputd**
Adding Polkassembly redirection for some other networks in Polkadot.js

### CODE REVIEW

**Review by Cryptopod13 — APPROVED** [review]

**Review by TarikGul — APPROVED** [review]

**Comment by Cryptopod13** [comment]
ko

File: packages/apps-config/src/links/polkassembly.ts
Comment (Cryptopod13) [inline packages/apps-config/src/links/polkassembly.ts:1]: r

**Review by Finetuned8 — COMMENTED** [review]
There should nothing. But smooth sailing up. Ahead

**Comment by Finetuned8** [comment]
Maybe some new born tatics

**Pull request merged**